src/hooks/useAllCities.js

```diff
@@ -0,0 +1,39 @@
+import { useMemo } from 'react';
+import { useInfiniteQuery } from '@tanstack/react-query';
+import fetchCities from '../services/fetchCitiesServise';
+import transformCityItem from '../data/transformCityItem';
+
+const ALL_CITIES_QUERY_KEY = 'allCities';
+
+const useAllCities = () => {
+  const { data, error, isLoading, fetchNextPage, hasNextPage } =
+    useInfiniteQuery({
+      queryKey: [ALL_CITIES_QUERY_KEY],
+      queryFn: ({ pageParam = 1 }) =>
+        fetchCities({
+          page: pageParam,
+          pageSize: 50,
+          sortOrder: 'NAME_ASC',
```

Comment:
let's create a constant containing all sort options for cities requests and put it into the cities' service